docs/7.x/cluster.md

```diff
@@ -410,6 +410,15 @@ drwxr-xr-x. 5 user user 4.0K Jan 14 09:08 packages
 You can execute the `./upgrade` script to upload and upgrade in one go,
 or you can upload the update and then execute `gravity upgrade` command.
 
+!!! note "Disabling existing functionality when upgrading"
+Currently, `gravity upgrade` will not uninstall existing applications that have been disabled in the upgrade manifest.
```

Comment:
I'd remove `currently` as it does not provide a useful time frame - when this is possible, I would just remove the warning.